helper/include/odp/helper/table.h

```diff
@@ -56,7 +56,7 @@
  *     actions and action meta-data describing what processing to be
  *     applied for the packets of the current flow, such as whether
  *     encryption/decryption is required on this packet, what kind of cipher
- *     algorithm should be chosed.
+ *     algorithm should be chosen
```

Comment:
The period at the end could still be retained.

Reply:
Added.